SasquatchMac/SasquatchMacSwift/AppDelegate.swift

```diff
@@ -43,8 +43,14 @@ class AppDelegate: NSObject, NSApplicationDelegate, MSCrashesDelegate, MSPushDel
     // Push Delegate.
     MSPush.setDelegate(self);
 
-    // Start AppCenter.
     MSAppCenter.setLogLevel(MSLogLevel.verbose)
+    // Set user id.
```

Comment:
We have a coding convention that there should be an empty line before any of comment in the code. Please add an empty line above.

Reply:
Fixed. Thank you.